README.md

```diff
@@ -1 +1,20 @@
-# hw01
\ No newline at end of file
+# hw01
+## biography
+### **basics**
+* Name: Hengle Li (Eric)
+* Year: first-year MAPSS
+* Concentration: Sociology
+
+#### _a few words_
+Hello, my name is Hengle Li. I also go by Eric, because there are so many various pronunciations of my first name, I don't want to bother anymore. 
+I am from China, and I went to college in UC Berkeley, as a sociology major. My research interests are social change, sociological theories, and political sociology. Bourdieu is my favorite, though I hate his writing style. After Bourdieu, I like Marx the best, because he's one of the few sociologists that know how to write properly for people to read their texts.  
+I have no programming experience before, so all these are like dark magic to me. Cheers!
+
+![my photo](https://scontent-ort2-2.xx.fbcdn.net/v/t1.0-9/22886269_1981378462118343_6779231979075832945_n.jpg?_nc_cat=107&_nc_oc=AQmiGwbjkz7ysKtsfUufMHsYhumTkmFJLz88NomnObfl46qNJsOmP4yuif76TDN3-WE&_nc_ht=scontent-ort2-2.xx&oh=a69a4d144d36e97f13c224e3c216040e&oe=5E37D896)
```

Comment:
Try using a relative path & upload your picture here. That will help simplify your code a bit.